### PR TITLE
Step 1: Create new style files based on v3 and remove dependency from WallectConnectModalTheme/WalletConnectModalThemeData

### PR DESCRIPTION
**Pull request by quetool**
**Context:**

Currently Web3ModalFlutter relies heavily on WalletConnectModalFlutter package. We want to change that and completely decouple them both.

Since Web3ModalFlutter is going to be restyled with v3 designs this is a first step towards that direction.

**Changes:**

- Created a new set of theme files exclusively for Web3ModalFlutter under `lib/theme` based on v3 style
- Removed every usage of `WalletConnectModalTheme/WalletConnectModalThemeData` as a first step to completely decouple `WalletConnectModalFlutter` from `Web3ModalFlutter`
- Left some TODOs that will be addressed the more we go forward with the decoupling of packages.

_Note:_ The PR points to an intermediate branch called feat/v3restyle as I will be adding breaking changes and we don’t want to break anything for devs that may be using the package. Once everything is done we will merge the intermediate branch with master

**Final goal:**

- To have a completely independent Web3Modal package with it's own v3 design style

### CODE REVIEW

File: example/sign/lib/pages/basic_page.dart [hunk 1]
```diff
@@ -37,7 +38,7 @@ class _BasicPageState extends State<BasicPage>
     with SingleTickerProviderStateMixin {
   bool _initialized = false;
 
-  bool _testnetOnly = false;
+  // bool _testnetOnly = false;
```
Comment (0xAsimetriq) [inline example/sign/lib/pages/basic_page.dart:41]: Is this comment needed / does it need TODO?

File: example/sign/lib/pages/w3m_page.dart [hunk 2]
```diff
@@ -21,39 +20,34 @@ class W3MPage extends StatefulWidget {
...
+  }
 
-    await _w3mService?.init();
-
-    _isConnected = widget.web3App.sessions.getAll().isNotEmpty;
+  Future<bool> _initializeService() async {
+    try {
+      _w3mService = W3MService(
+        web3App: widget.web3App,
+        recommendedWalletIds: {
```
Comment (0xAsimetriq) [inline example/sign/lib/pages/w3m_page.dart:38]: Ideally these should be dynamic i.e. first 3 (or more) wallets returned from api

File: lib/widgets/w3m_account_button.dart [hunk 3]
```diff
@@ -30,21 +30,20 @@ class _W3MAccountButtonState extends State<W3MAccountButton> {
...
 
     return Container(
       padding: const EdgeInsets.all(4.0),
       height: 40,
       decoration: BoxDecoration(
-        color: themeData.background200,
+        color: themeData.colors.background200,
         borderRadius: const BorderRadius.all(
           Radius.circular(
             10, //themeData.radiusXS,
```
Comment (0xAsimetriq) [inline lib/widgets/w3m_account_button.dart:42]: Does this comment need a todo as well?

**Review by 0xAsimetriq — COMMENTED** [review]
Great start!

File: example/sign/lib/pages/w3m_page.dart (hunk 2 above)
Comment (quetool) [reply example/sign/lib/pages/w3m_page.dart:38]: `recommendedWalletIds` is used to give priority to wallets of choice. Check here https://github.com/WalletConnect/WalletConnectModalFlutter#excluding-recommending-and-including-wallets

File: lib/widgets/w3m_account_button.dart (hunk 3 above)
Comment (quetool) [reply lib/widgets/w3m_account_button.dart:42]: I'll remove it!

File: example/sign/lib/pages/basic_page.dart (hunk 1 above)
Comment (quetool) [reply example/sign/lib/pages/basic_page.dart:41]: I'll remove it!

**Review by 0xAsimetriq — APPROVED** [review]
LGTM!

**Pull request merged**